### ISSUE TITLE
Remove magrittr dependency in the R documentation for cb.gblinear.history

### ISSUE THREAD

**Issue by ReeceGoding** [dmlc/xgboost#6852]
The documentation for `cb.gblinear.history`, generated [here](https://github.com/dmlc/xgboost/blob/master/R-package/R/callbacks.R), has a dependency on magrittr. While this is not a particular obscure library, it seems strange that we assume familiarity with it. Furthermore, the six calls to the library that I've found in the documentation all appear to be trivial. If I'm not mistaken, they're all of the form `xgb.gblinear.history(foo) %>% matplot(type = 'l')`. This should be trivial to express without magrittr. In fact, earlier on in the documentation, what I believe to be one such equivalent call appears:
```R
coef_path <- xgb.gblinear.history(bst)
matplot(coef_path, type = 'l')
```
So why not just get rid of the magrittr dependency in the documentation, even if just for consistency's sake? I don't believe that it's used in any of our other documentation.

As a final point, I believe that the next version of R is getting its own pipe operator, so even if we do want to use pipes in our documentation, we soon won't need magrittr's.

I should have a pull request ready for this tonight. It seems like a trivial job to do.

**Comment by hcho3** [comment]
Thanks! We'll make sure to review your pull request.

**Comment by trivialfis** [comment]
Closed by #6855
